### ISSUE TITLE
Stop with error if file size exceeds limit

### ISSUE THREAD

**Issue by smith** [nodejitsu/node-cloudfiles#1]
From the devguide:

"If users attempt to upload more that 5GB with this method, the server will close the TCP/IP connection after 5GB and purge the customer data from the system. Users must take responsibility to ensure the data they transfer will be less than 5GB or to split it into 5GB chunks each in its own storage Object."

Make the client preemptively throw an error if the file is to large.

Don't know if this is implemented or not on this implementation, but I also opened http://github.com/rackspace/ruby-cloudfiles/issues/issue/10

**Comment by Marak** [comment]
Woah, that is good to know. Thanks for the heads up. We will be trying to push v0.1.0 of this sometime over the weekend. If anything we will make sure to put a warning in the README.

**Comment by smith** [comment]
Yes I was getting a 413 after waiting a long time to upload a 7gig file today, and rackspace chat support told me about it.
